src/condor_status.V6/prettyPrint.cpp

```diff
@@ -565,12 +588,23 @@ const char * const stateCompact_PrintFormat = "SELECT\n"
 "GROUP BY Machine\n"
 "SUMMARY STANDARD\n";
 
-void PrettyPrinter::ppSetStateCols (int width)
+const char * const startdState_PrintFormat = "SELECT\n"
+	ATTR_NAME                         " WIDTH -34 FIT PRINTF %s OR ??\n"
+	ATTR_CPUS                " AS Cpu           OR ??\n"
```

Comment:
This and the `ActvtyTime column no longer have widths being set. Is this intentional?

Reply:
yes, could not get it to work properly with widths.